helm/tailing-sidecar-operator/README.md

```diff
@@ -0,0 +1,37 @@
+# Tailing Sidecar Operator
+
+![Project Status](https://img.shields.io/badge/status-alpha-important?style=for-the-badge)
+
+[Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar/tree/main/operator) makes it easy to add
```

Comment:
It seems that this link could be relative as it is inside repository.

Reply:
Done in https://github.com/SumoLogic/tailing-sidecar/pull/82/commits/0c49d6e694b1620fa97f12b036bba67408b4bfd8